src/antsibull_changelog/ansible.py

```diff
@@ -41,48 +23,66 @@
 PLUGIN_EXCEPTIONS = (("cache", "base.py"), ("module", "async_wrapper.py"))
 
 
+@cache
 def get_documentable_plugins() -> tuple[str, ...]:
     """
     Retrieve plugin types that can be documented.
     """
-    if HAS_ANSIBLE_CONSTANTS:
-        return C.DOCUMENTABLE_PLUGINS
-    return (
-        "become",
-        "cache",
-        "callback",
-        "cliconf",
-        "connection",
-        "httpapi",
-        "inventory",
-        "lookup",
-        "netconf",
-        "shell",
-        "vars",
-        "module",
-        "strategy",
-    )
-
+    try:
+        # We import from ansible locally since importing it is rather slow
+        from ansible import constants as C  # pylint: disable=import-outside-toplevel
 
+        return C.DOCUMENTABLE_PLUGINS
+    except ImportError:
+        return (
+            "become",
+            "cache",
+            "callback",
+            "cliconf",
+            "connection",
+            "httpapi",
+            "inventory",
+            "lookup",
+            "netconf",
+            "shell",
+            "vars",
+            "module",
+            "strategy",
+        )
+
+
+@cache
 def get_documentable_objects() -> tuple[str, ...]:
     """
     Retrieve object types that can be documented.
     """
-    if not HAS_ANSIBLE_RELEASE:
-        return ()
-    if packaging.version.Version(
-        ansible_release.__version__
-    ) < packaging.version.Version("2.11.0"):
+    try:
+        # We import from ansible locally since importing it is rather slow
+        # pylint: disable-next=import-outside-toplevel
+        from ansible import release as ansible_release
+
+        if packaging.version.Version(
+            ansible_release.__version__
+        ) < packaging.version.Version("2.11.0"):
+            return ()
+        return ("role",)
+    except ImportError:
         return ()
-    return ("role",)
 
 
+@cache
 def get_ansible_release() -> tuple[str, str]:
     """
     Retrieve current version and codename of Ansible.
 
     :return: Tuple with version and codename
     """
-    if not HAS_ANSIBLE_RELEASE:
+    try:
+        # We import from ansible locally since importing it is rather slow
+        # pylint: disable-next=import-outside-toplevel
+        from ansible import release as ansible_release
+
+        return ansible_release.__version__, ansible_release.__codename__
+    except ImportError:
+        # pylint: disable-next=raise-missing-from
         raise ValueError("Cannot import ansible.release")
```

Comment:
```suggestion
    except ImportError as exc:
        # pylint: disable-next=raise-missing-from
        raise ValueError("Cannot import ansible.release") from exc
```

would prevent exception while handling exception error.